kikuchipy/indexing/static_dictionary_indexing.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019-2020 The kikuchipy developers
+#
+# This file is part of kikuchipy.
+#
+# kikuchipy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kikuchipy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kikuchipy. If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Union, List
+import dask.array as da
+from dask.diagnostics import ProgressBar
+import numpy as np
+from h5py import File
+
+from orix.quaternion.rotation import Rotation
+from orix.crystal_map.crystal_map import CrystalMap, PhaseList, Phase
+
+from kikuchipy.indexing import pattern_match
+
+from kikuchipy.indexing.osm import orientation_similarity_map
+from kikuchipy.indexing.similarity_metrics import (
+    SimilarityMetric,
+    SIMILARITY_METRICS,
+)
+
+from kikuchipy.indexing.merge_crystalmaps import merge_crystalmaps
+
+
+class StaticDictionaryIndexing:
+    """Indexing against pre-computed dictionaries from EMsoft's EMEBSD.f90 program"""
```

Comment:
I would change this explanation, as we're on track to computing this dictionary ourselves.

Reply:
Wording is now changed, reflecting the future case where we can generate this static dictionary ourselves.